ducktape/templates/report/report.html

```diff
@@ -62,7 +70,7 @@
       
       var TestRow = React.createClass({
         render: function() {
-          var className = this.props.test.test_result + "Class"
+          var className = this.props.test.test_result
```

Comment:
Nit: The `Detail` column doesn't point to anything valid when a test has been ignored.